FakeItEasy.PrepareRelease/GitHubHelper.cs

```diff
@@ -0,0 +1,117 @@
+﻿namespace FakeItEasy.PrepareRelease;
```

Comment:
I'm not sure if we want to use file-scoped namespaces here. On one hand, it reduces indentation, which is nice; on the other hand, it's not consistent with the rest of the code, and we probably don't want to touch every file in the project just to use this feature...

Reply:
I am confused. You appear to be using on even though you're not sure?
TBH, I don't care one way or the other about the indentation. Whichever you prefer is fine with me.

Reply:
What I mean is that I introduced the use of file-scoped namespaces here without regard for consistency with the rest of the code.

For these tools, it doesn't really matter, since they're not really part of the "main" project. But in the future we'll need to decide what to do in FakeItEasy itself: continue using old-school namespaces with braces, or start using file-scoped namespaces? And if we do, do we care about consistency between files?

Anyway, we don't need to decide that now. If you don't mind that I used FSN here, that's good enough for me.

Reply:
Ah, thanks. I don't mind here. 

Agreed, we should eventually decide about file-scoped namespaces!